### DISCUSSION TITLE
When will Harbor2.5.0 with cosign be released?

### DISCUSSION THREAD

**Discussion by neblen** [goharbor/harbor#16446] [Q&A] [answered]
Hello everyone! I read the roadmap of harbor.The roadmap doesn‘t introduce about Harbor 2.5.0.I konw that harbor will integrate cosign in harbor2.5.0.So I want to know an approximate time.Thank you very much!

**Comment by wy65701436** [comment, marked as answer]
It's GAed today, check it out.
